src/getHumanAge.test.js

```diff
@@ -7,4 +7,62 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return array of numbers`, () => {
+    const catAgeMock = null;
+    const dogAgeMock = null;
+    const result = getHumanAge(catAgeMock, dogAgeMock);
+
+    expect(Array.isArray(result)).toBeTruthy();
+    expect(result).toHaveLength(2);
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should return 0 humanAge if input value less than 15 years', () => {
```

Comment:
The same like in previous task
 test(`should return 0 for cat and dog when
    they are less than 15 years old`, () => { // both are less than 1 year old
    expect(getHumanAge(5, 5))
      .toEqual([0, 0]);
  });